src/python/grpcio_tests/tests/reflection/_reflection_servicer_test.py

```diff
@@ -50,6 +50,7 @@
 _SERVICE_NAMES = ('Angstrom', 'Bohr', 'Curie', 'Dyson', 'Einstein', 'Feynman',
                   'Galilei')
 _EMPTY_EXTENSIONS_SYMBOL_NAME = 'grpc.testing.proto2.EmptyWithExtensions'
+_EMPTY_EXTENSIONS_NUMBERS = (124, 125, 126, 127, 128)
```

Comment:
Include the ostensibly-optional-but-always-a-good-idea last-position comma after `128`.